src/main/java/uk/co/ramp/covid/simulation/population/Population.java

```diff
@@ -173,7 +173,7 @@ public void populateHouseholds() {
 
         createPopulation(adultIndex, pensionerIndex, childIndex, infantIndex);
 
-        assert householdAllocationPossible(adultIndex, pensionerIndex, childIndex, infantIndex)
+        assert householdAllocationImpossible(adultIndex, pensionerIndex, childIndex, infantIndex)
```

Comment:
I think we want to assert it is possible? Rather than impossible? The improvedErrorHandling branch removes this anyway so it's probably not a big deal

Reply:
Renamed boolean to 'possible' and reversed the conditions.